src/Bicep.LangServer/Deploy/CredentialFromTokenAndTimeStamp.cs

```diff
@@ -9,9 +9,16 @@ public class CredentialFromTokenAndTimeStamp : TokenCredential
     {
```

Comment:
See Conversation tab for the UI changes.  NOTE: I've tested all deployment scopes with both the Deploy Bicep File... menu and also the deployment pane.  I've also tested on ChinaCloud, but deployment pane doesn't currently succeed with sovereign clouds (https://github.com/Azure/bicep/issues/14834)